wiso.js

```diff
@@ -36,19 +36,21 @@
 
 
 function detectWeb(doc, url) {
-	if (url.search(/START=A[246]0/) !== -1) {
+	if (url.search(/document/) !== -1) {
 		if( ZU.xpath(doc, '//a[contains(@class, "boxExport")]').length>0 ) {
 			//single item --> generic fallback = journalArticle
 			return "journalArticle";
 		}
-	} else if ( ZU.xpath(doc, '//a/span[contains(@class, "boxHeader")]').length>0 ) {
+	} else if (getSearchResults(doc, true)) {
 		return "multiple";
 	}
 }
 
 
 function scrape(doc, url) {
-	var risUrl = url+"&OHNE_SAVE=1&T_TEMPLATE=ris";
+	var permaUrl = ZU.xpathText(doc, '//pre[strong]/text()');
+	var docUid = permaUrl.substr(permaUrl.lastIndexOf('/')+1);
+	var risUrl = "https://www.wiso-net.de/stream/exportDocuments?docUids=" + docUid + "&dbShortcut=&query=&source=Document&format=Citavi";
```

Comment:
Use relative URL ("/stream/....") and split the line into two, since it's too long. Also, you can append the docUids to the end of the URL, position does not matter.